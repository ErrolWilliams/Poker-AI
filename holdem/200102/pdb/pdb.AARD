AARD      981833712  6  6 f   -     -     -         1000    0    0 
AARD      981833746  6  5 f   -     -     -         1000    0    0 
AARD      981833768  6  3 Q   -     -     -         1000    0    0 
