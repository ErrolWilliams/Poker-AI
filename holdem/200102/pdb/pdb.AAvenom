AAvenom   982436238  7  7 cc  f     -     -         1000   30    0 
AAvenom   982436361 10  6 f   -     -     -          970    0    0 
AAvenom   982436442 10  5 cc  ccc   krc   b          970  160  550 8c 7d
AAvenom   982436542 10  4 rr  c     c     c         1360   90  390 Qd Qh
AAvenom   982436625 10  3 f   -     -     -         1660    0    0 
AAvenom   982436696 10  2 Bc  k     kc    bc        1660   80  225 7h As
AAvenom   982436803 10  1 Bf  -     -     -         1805    5    0 
AAvenom   982436900 10 10 f   -     -     -         1800    0    0 
AAvenom   982436984 10  9 f   -     -     -         1800    0    0 
AAvenom   982437042 10  8 f   -     -     -         1800    0    0 
AAvenom   982437125 10  7 f   -     -     -         1800    0    0 
AAvenom   982437172  9  5 f   -     -     -         1800    0    0 
AAvenom   982437226 10  5 c   b     b     k         1800   40  125 4d Ah
AAvenom   982437300 10  5 f   -     -     -         1885    0    0 
AAvenom   982437340  9  3 c   k     k     k         1885   10   10 8h 9c
AAvenom   982437405  9  2 Bk  kf    -     -         1885   10    0 
AAvenom   982437459  9  1 Brc kc    krc   k         1875  120  420 Qd Ah
AAvenom   982437542 10 10 c   c     c     k         2175   50    0 Qd Jc
AAvenom   982437658 10  3 Q   -     -     -         2125    0    0 
AAvenom   982438193 10  8 f   -     -     -         1295    0    0 
AAvenom   982438282 10  7 f   -     -     -         1295    0    0 
AAvenom   982438348 10  6 f   -     -     -         1295    0    0 
AAvenom   982438434 10  6 r   b     k     kf        1295   30    0 
AAvenom   982438495 10  4 c   k     b     bcc       1265  110    0 5h 8h
AAvenom   982438614 10  3 fQ  -     -     -         1155    0    0 
AAvenom   982444015  9  2 Bf  -     -     -         1155   10    0 
AAvenom   982444152  6  1 Bf  -     -     -         1145    5    0 
AAvenom   982444203  7  7 f   -     -     -         1140    0    0 
AAvenom   982444287  8  7 c   rr    c     c         1140  100    0 7c 3c
AAvenom   982444338  8  6 rr  rc    c     f         1040   90    0 
AAvenom   982444407  8  5 fQ  -     -     -          950    0    0 
AAvenom   982796452 10  7 cc  c     c     c          795   90    0 4d 4h
AAvenom   982796528  9  6 f   -     -     -          705    0    0 
AAvenom   982796592 10  5 rc  kc    kr    b          705  100  390 6s 6h
AAvenom   982796752  9  4 c   k     c     c          995   50    0 Ad Qh
AAvenom   982796813  9  3 c   kf    -     -          945   10    0 
AAvenom   982796871 10  2 Bk  b     b     b          935   60  120 
AAvenom   982796927 10  1 Bf  -     -     -          995    5    0 
AAvenom   982796999 10 10 f   -     -     -          990    0    0 
AAvenom   982797060 10  9 f   -     -     -          990    0    0 
AAvenom   982797147 10  8 c   k     r     c          990   80  150 5c 5s
AAvenom   982797222 10  3 Q   -     -     -         1060    0    0 
AAvenom   982980963 10  6 f   -     -     -         1060    0    0 
AAvenom   982981025 10  5 f   -     -     -         1060    0    0 
AAvenom   982981121 10  4 f   -     -     -         1060    0    0 
AAvenom   982981183 10  3 c   kf    -     -         1060   10    0 
AAvenom   982981294 10  2 Bk  b     kf    -         1050   20    0 
AAvenom   982981359  9  1 Bc  b     b     b         1030   60    0 Jh Ks
AAvenom   982981421 10 10 c   b     c     b          970   60  190 2s 2c
AAvenom   982981491  8  7 c   c     c     f         1100   40    0 
AAvenom   982981566 10  3 Q   -     -     -         1060    0    0 
