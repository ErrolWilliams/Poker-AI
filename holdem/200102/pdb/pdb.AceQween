AceQween  982377016  8  5 f   -     -     -         1000    0    0 
AceQween  982377054  8  4 f   -     -     -         1000    0    0 
AceQween  982377076  8  3 rc  r     b     b         1000  100  255 As Ts
AceQween  982377122  8  2 Bc  kf    -     -         1155   20    0 
AceQween  982377181  8  1 Bf  -     -     -         1135    5    0 
AceQween  982377211  8  8 rc  cc    f     -         1130   80    0 
AceQween  982377268 10  9 f   -     -     -         1050    0    0 
AceQween  982377354 10  8 f   -     -     -         1050    0    0 
AceQween  982377425  9  7 f   -     -     -         1050    0    0 
AceQween  982377528  9  5 f   -     -     -         1050    0    0 
AceQween  982377579  9  5 f   -     -     -         1050    0    0 
AceQween  982377630  9  4 f   -     -     -         1050    0    0 
AceQween  982377672  9  3 f   -     -     -         1050    0    0 
AceQween  982377723  9  2 Bk  k     bc    bc        1050   90    0 Jh 5d
AceQween  982377766  9  1 Br  kf    -     -          960   30    0 
AceQween  982377826  9  9 f   -     -     -          930    0    0 
AceQween  982377889  9  8 f   -     -     -          930    0    0 
AceQween  982377940  9  7 f   -     -     -          930    0    0 
AceQween  982377996  8  6 c   c     f     -          930   30    0 
AceQween  982378065  7  4 f   -     -     -          900    0    0 
AceQween  982378128  7  4 c   c     b     brc        900  120  290 Qd 9d
AceQween  982378203  7  3 f   -     -     -         1070    0    0 
AceQween  982378270  7  2 Bk  kf    -     -         1070   10    0 
AceQween  982378331  7  1 Bc  b     b     b         1060   60  140 
AceQween  982378369  7  7 f   -     -     -         1140    0    0 
AceQween  982378393  7  6 f   -     -     -         1140    0    0 
AceQween  982378468  7  5 f   -     -     -         1140    0    0 
AceQween  982378513  7  4 f   -     -     -         1140    0    0 
AceQween  982378570  7  3 c   k     b     kc        1140   50    0 9s 7s
AceQween  982378630  6  1 Bc  b     bc    kc        1090   90   90 Ad 7s
AceQween  982378666  6  1 Bf  -     -     -         1090    5    0 
AceQween  982378778  5  5 f   -     -     -         1085    0    0 
AceQween  982378810  5  4 f   -     -     -         1085    0    0 
AceQween  982378855  5  3 f   -     -     -         1085    0    0 
AceQween  982378888  5  2 Bk  br    b     b         1085   80  170 3c Kh
AceQween  982378939  5  1 Bf  -     -     -         1175    5    0 
AceQween  982378976  4  4 f   -     -     -         1170    0    0 
AceQween  982379037  4  3 rc  rr    b     k         1170   90  190 6h 6d
AceQween  982379079  4  2 Bf  -     -     -         1270   10    0 
AceQween  982379108  4  1 Bf  Q     -     -         1260    5    0 
AceQween  982488680  9  5 f   -     -     -         1255    0    0 
AceQween  982488724  9  4 f   -     -     -         1255    0    0 
AceQween  982488806  9  3 f   -     -     -         1255    0    0 
AceQween  982488854  9  2 Bk  k     k     k         1255   10    0 8d Kh
AceQween  982488958  8  1 Bf  -     -     -         1245    5    0 
AceQween  982489004  9  8 c   b     c     r         1240   80  240 Tc 9c
AceQween  982489116  8  7 cc  r     b     k         1400   60    0 Js Th
AceQween  982489194  8  6 f   -     -     -         1340    0    0 
AceQween  982489282  8  5 f   -     -     -         1340    0    0 
AceQween  982489332  7  3 f   -     -     -         1340    0    0 
AceQween  982489384  8  2 Bk  k     k     b         1340   30  110 Tc Jd
AceQween  982489434  8  1 Br  kc    kc    kf        1420   50    0 
AceQween  982489518  8  8 c   c     c     c         1370   60    0 9h 9s
AceQween  982489555  8  7 f   -     -     -         1310    0    0 
AceQween  982489634  8  6 c   c     k     b         1310   40   90 
AceQween  982489692  8  5 cc  k     b     k         1360   40    0 Ah 2c
AceQween  982489746  8  4 f   -     -     -         1320    0    0 
AceQween  982489816  8  3 c   f     -     -         1320   10    0 
AceQween  982489860  8  2 Brc kf    -     -         1310   40    0 
AceQween  982489936  8  1 Bf  -     -     -         1270    5    0 
AceQween  982489987  8  8 r   bc    c     c         1265   80    0 Ac 9h
AceQween  982490076  8  7 f   -     -     -         1185    0    0 
AceQween  982490186  8  6 f   -     -     -         1185    0    0 
AceQween  982490322  7  5 c   b     b     k         1185   40    0 9h 7h
AceQween  982490376  7  4 f   -     -     -         1145    0    0 
AceQween  982490421  7  3 cc  kc    kc    b         1145   70  270 
AceQween  982490536  6  2 Bc  bc    kc    kc        1345   80    0 7c Ts
AceQween  982490672  6  1 Bf  -     -     -         1265    5    0 
AceQween  982490728  6  6 cc  r     b     -         1260   60  155 
AceQween  982490867  5  4 rc  k     f     -         1355   30    0 
AceQween  982490935  6  4 f   -     -     -         1325    0    0 
AceQween  982490975  6  3 f   -     Q     -         1325    0    0 
AceQween  982621370 10  5 cc  kc    kf    -         1325   30    0 
AceQween  982621501  8  3 f   -     -     -         1295    0    0 
AceQween  982621543  9  3 f   -     -     -         1295    0    0 
AceQween  982621624 10  2 Bk  kf    -     -         1280   10    0 
AceQween  982621689 10  1 Bc  kf    -     -         1250   20    0 
AceQween  982621751 10 10 f   -     -     -         1230    0    0 
AceQween  982621835  8  7 r   b     r     b         1210   90  305 As Ac
AceQween  982621907  8  6 f   -     -     -         1405    0    0 
AceQween  982621997  8  5 f   -     -     -         1400    0    0 
AceQween  982622072  6  3 c   brc   kcc   kcc       1545  210    0 8d 8s
AceQween  982622206  7  2 Bf  -     -     -         1335   10    0 
AceQween  982622292  8  1 Bc  k     brc   bc        1325  130  270 5s 7h
AceQween  982622365 10 10 c   f     -     -         1365   20    0 
AceQween  982622469 10  9 f   -     -     -         1215    0    0 
AceQween  982622565 10  8 f   -     -     -         1200    0    0 
AceQween  982622617 10  7 cc  kf    -     -         1200   20    0 
AceQween  982622672  9  5 f   -     -     -         1180    0    0 
AceQween  982622729  8  3 f   -     -     -         1180    0    0 
AceQween  982622773  8  2 Br  b     kf    -         1180   30    0 
AceQween  982622824  8  1 Bc  k     kf    -         1150   10    0 
AceQween  982622903  9  9 f   -     -     -         1140    0    0 
AceQween  982622984 10  9 c   f     -     -         1140   20    0 
AceQween  982623047 10  8 f   -     -     -         1110    0    0 
AceQween  982623093 10  8 f   -     -     -         1070    0    0 
AceQween  982623181  9  6 f   -     -     -         1065    0    0 
AceQween  982623235  9  5 f   -     -     -         1065    0    0 
AceQween  982623281  9  4 f   -     -     -         1070    0    0 
AceQween  982623348  9  3 f   -     -     -         1080    0    0 
AceQween  982623406  9  2 Bk  kc    kc    kf        1040   40    0 
AceQween  982623480 10  1 Brc br    k     bc        1000  110    0 Ts Tc
AceQween  982623577 10 10 f   -     -     -          890    0    0 
AceQween  982623647 10  9 f   -     -     -          890    0    0 
AceQween  982623739 10  8 f   -     -     -          890    0    0 
AceQween  982623800 10  7 f   -     -     -          935    0    0 
AceQween  982623868 10  6 f   -     -     -          895    0    0 
AceQween  982623919 10  3 Q   -     -     -          895    0    0 
