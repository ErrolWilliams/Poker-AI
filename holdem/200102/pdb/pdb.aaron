aaron     982473761 10  4 f   -     -     -         1000    0    0 
aaron     982473845  9  2 Bk  k     k     f         1000   10    0 
aaron     982473897 10  1 BQ  -     -     -          990    5    0 
