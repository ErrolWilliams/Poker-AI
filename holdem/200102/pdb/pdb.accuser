accuser   981947318 10  8 KQ  -     -     -          905    0    0 
