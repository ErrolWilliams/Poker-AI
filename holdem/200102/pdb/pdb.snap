snap      983229223  4  1 BKQ -     -     -         1000    5    0 
